test/test_torch.py

```diff
@@ -19996,9 +20036,9 @@ def test_split_view(self, device):
             v[0, 0] = idx + 1
             self.assertEqual(t[idx, 0], v[0, 0])
 
-    def test_movedim_view(self, device):
+    def _test_movedim_swapdim_view(self, device, op):
```

Comment:
We have test_tranpose_view, already, which you can reuse with the above technique.